src/main/java/com/codecool/dungeoncrawl/data/GameMap.java

```diff
@@ -14,7 +14,7 @@ public class GameMap {
 
     private Skeleton skeleton;
 
-    private Ghost ghost;
+    private Giant giant;
```

Comment:
I mentioned DI is breaking, but also Open Closed is breaking, as you introduce more types of enemies you have to make that many changes in your GameMap. It would be better if you didn't have to make these changes.